hazelcast/src/main/java/com/hazelcast/spi/impl/executionservice/impl/ExecutionServiceImpl.java

```diff
@@ -243,6 +248,16 @@ public void executeDurable(String name, Runnable command) {
     }
 
     @Override
+    public ScheduledFuture<?> scheduleDurable(String name, Runnable command, long delay, TimeUnit unit) {
+        return getDurableTaskScheduler(name).schedule(command, delay, unit);
```

Comment:
So we are delegating the tasks to durable-executor. Imagine this scenario:
user obtains durable-executor and scheduled-executor with the same name
```
instance.getDurableExecutorService("foo");
instance.getScheduledExecutorService("foo");
```
and uses both of them accordingly. at some time s/he decides to shutdown the scheduled-executor, which will shutdown the underlying durable-executor.

Won't this be a problem ? 

As a solution, we can add some prefix to the name like `hz:impl:scheduledExecutorService` while obtaining the durable-executor.